src/org/sosy_lab/common/log/TestLogManager.java

```diff
@@ -83,25 +82,46 @@ public void logf(Level pPriority, String pFormat, Object... pArgs) {
   }
 
   @Override
-  public void logUserException(Level pPriority, Throwable pE, @Nullable String pAdditionalMessage) {
+  public void logUserException(Level pPriority, Throwable pE, Object... pArgs) {
     checkNotNull(pPriority);
     checkNotNull(pE);
+    checkNotNull(pArgs);
   }
 
   @Override
-  public void logDebugException(Throwable pE, @Nullable String pAdditionalMessage) {
+  public void logfUserException(Level pPriority, Throwable pE, String pFormat, Object... pArgs) {
+    checkNotNull(pPriority);
     checkNotNull(pE);
+    checkNotNull(pFormat);
+    checkNotNull(pArgs);
```

Comment:
This and similar methods should get the same checks as `logf(Level, String, Object...)`, probably by refactoring them out into a separate method.

Reply:
Also Done